discordbot/stocks/technical_analysis/ta_menu.py

```diff
@@ -0,0 +1,916 @@
+import asyncio
+import discord
+import discord_components
+import config_discordbot as cfg
+import helpers
+from datetime import datetime, timedelta
+
+from stocks.technical_analysis.ema import ema_command
+from stocks.technical_analysis.sma import sma_command
+from stocks.technical_analysis.wma import wma_command
+from stocks.technical_analysis.hma import hma_command
+from stocks.technical_analysis.zlma import zlma_command
+from stocks.technical_analysis.cci import cci_command
+from stocks.technical_analysis.macd import macd_command
+from stocks.technical_analysis.rsi import rsi_command
+from stocks.technical_analysis.stoch import stoch_command
+from stocks.technical_analysis.fisher import fisher_command
+from stocks.technical_analysis.cg import cg_command
+from stocks.technical_analysis.adx import adx_command
+from stocks.technical_analysis.aroon import aroon_command
+from stocks.technical_analysis.bbands import bbands_command
+from stocks.technical_analysis.donchian import donchian_command
+from stocks.technical_analysis.kc import kc_command
+from stocks.technical_analysis.ad import ad_command
+from stocks.technical_analysis.adosc import adosc_command
+from stocks.technical_analysis.obv import obv_command
+from stocks.technical_analysis.fib import fib_command
+from stocks.technical_analysis.view import view_command
+from stocks.technical_analysis.summary import summary_command
+from stocks.technical_analysis.recom import recom_command
+from discordbot import gst_bot
+
+
+class TechnicalAnalysisCommands(discord.ext.commands.Cog):
+    """Technical Analysis menu."""
+
+    def __init__(self, bot: discord.ext.commands.Bot):
+        self.bot = bot
+
+    @discord.ext.commands.command(name="stocks.ta.ema")
+    async def ema(
+        self,
+        ctx: discord.ext.commands.Context,
+        ticker="",
+        window="",
+        offset="",
+        start="",
+        end="",
+    ):
+        """Displays chart with ema of a given stock and sends it
+
+        Parameters
+        -----------
+        ticker: str
+            ticker, -h or help
+        window: int
+            window length. Default: 20, 50
+        offset: int
+            offset. Default: 0
+        start:
+            date (in date format for start date)
+        end:
+            date (in date format for end date)
+        """
+
+        await ema_command(ctx, ticker, window, offset, start, end)
+
+    @discord.ext.commands.command(name="stocks.ta.sma")
+    async def sma(
+        self,
+        ctx: discord.ext.commands.Context,
+        ticker="",
+        window="",
+        offset="",
+        start="",
+        end="",
+    ):
+        """Displays chart with sma of a given stock and sends it
```

Comment:
I would rephrase all these sentences to:
`"""Displays chart with simple moving average [Yahoo Finance]"""`
i.e.:
* Avoid the `and sends it`
* Be more explicit what it does so when a user does !help they know what this command does, 
* Refer to the data source between brackets []